app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/SavingThrow.java

```diff
@@ -0,0 +1,43 @@
+package yacs5e.ptpthingers.com.yacs5e_app;
+
+class SavingThrow {
+
+    Boolean proficient; // TRUE if proficient, FALSE if not
+
+    // Advantages and disadvantages:
+    // 0 for nothing
+    // 1 for enabling
+    // -1 for locking the value in disabled state
+    Byte advantage;
+    Byte disadvantage;
+
+    public SavingThrow() {
+        proficient = Boolean.FALSE;
+        advantage = 0;
+        disadvantage = 0;
+    }
+
+    public Boolean getProficient() {
+        return proficient;
```

Comment:
Proposition:
Maybe some function that takes a list as an argument and count the resulting advantage?

Reply:
I don't like it ;x